docs/csv-description.md

```diff
@@ -19,6 +19,12 @@ The Koku Metrics Operator (`koku-metrics-operator`) collects the metrics require
 * PersistentVolumeClaim (PVC) configuration: The KokuMetricsConfig CR can accept a PVC definition and the operator will create and mount the PVC. If one is not provided, a default PVC will be created.
 * Restricted network installation: this operator can function on a restricted network. In this mode, the operator stores the packaged reports for manual retrieval.
 
+## New in v3.3.2:
+* Leader election timing is now configurable via these environment variables:
+  * `LEADER_ELECTION_LEASE_DURATION` - sets the lease timeout duration
+  * `LEADER_ELECTION_RENEW_DEADLINE` - specifies the deadline to renew the lease
+  * `LEADER_ELECTION_RETRY_PERIOD` - defines the interval between retry attempts
```

Comment:
So how does a customer change these? That should be documented somewhere.

Reply:
Thanks for catching this, updated the docs to include guidance.